server/auto_merge.go

```diff
@@ -105,19 +107,41 @@ func (s *Server) AutoMergePR() error {
 			MergeMethod: "squash",
 		}
 
+		if translationPr {
+			opt.MergeMethod = "merge"
+		}
+
 		merged, _, err := s.GithubClient.PullRequests.Merge(ctx, pr.RepoOwner, pr.RepoName, pr.Number, "Automatic Merge", opt)
 		if err != nil {
 			errMsg := fmt.Sprintf("Error while trying to automerge the PR\nErr %s", err.Error())
 			if err = s.sendGitHubComment(ctx, pr.RepoOwner, pr.RepoName, pr.Number, errMsg); err != nil {
 				mlog.Warn("Error while commenting", mlog.Err(err))
 			}
+			if err = s.removeTranslationLabel(ctx, pr); err != nil {
+				mlog.Warn("Error while removing translation label", mlog.Err(err))
+			}
+			if err = s.sendTranslationWebhookMessage(ctx, pr, s.Config.TranslationsMergeFailureMessage); err != nil {
+				mlog.Warn("Error while sending failure message to mattermost", mlog.Err(err))
+			}
 			continue
 		}
 
 		msg = fmt.Sprintf("%s\nSHA: %s", merged.GetMessage(), merged.GetSHA())
 		if err = s.sendGitHubComment(ctx, pr.RepoOwner, pr.RepoName, pr.Number, msg); err != nil {
 			mlog.Warn("Error while commenting", mlog.Err(err))
 		}
+
+		if translationPr {
+			if err = s.removeTranslationLabel(ctx, pr); err != nil {
+				mlog.Warn("Error while removing translation label", mlog.Err(err))
+			}
+			if err = s.sendTranslationWebhookMessage(ctx, pr, s.Config.TranslationsMergedMessage); err != nil {
+				mlog.Warn("Error while sending translations merged message to mattermost", mlog.Err(err))
+			}
+			if err = s.sendGitHubComment(ctx, pr.RepoOwner, pr.RepoName, pr.Number, "/cherry-pick cloud"); err != nil {
```

Comment:
Can you help explain why we should `/cherry-pick cloud` as part of auto merging a translation PR? I could see incoming translation changes depending on `master` and possibly not yet being compatible with `cloud`.

Reply:
It is one of the acceptance criteria of the ticket. And yes automatically cherry picking into cloud may have issues. @spirosoik will cherry-picking into cloud branch may cause issues?

Reply:
@lieut-data Discussed with Spiros and `/cherry-pick cloud` will create a PR against cloud branch and do not automatically merge it. It will still need manual review and merge operation. So reviewers can fix PR when needed. If there is no issue they may approve and merge.

Reply:
@pfltdv & @spirosoik, I might be misunderstanding, but it's still not clear to me why weblate changes should get into cloud through a cherry pick instead of just the regular cloud branch changes. Reviewers won't be in a position to fix functional issues when, for example, the weblate changes introduce new variable dependencies from a commit that's still only in master and won't be in cloud until that branch is updated.

Or am I fundamentally misunderstanding?

Reply:
There is an existing process to manually cherry-pick translation files to the cloud. Before every cloud release, release manager ensures that commits related to translation PRs are cherry-picked into the cloud branch. Sample PRs:
https://github.com/mattermost/mattermost-server/pull/19372
https://github.com/mattermost/mattermost-server/pull/19639

Reply:
@amyblais, thoughts on changing this process to stop cherry-picking? I imagine we do the cherry-picks "early" in order to try to make cloud as "translated" as possible. I'm worried that this adds unnecessary overhead, especially if we accidentally get ahead of master by mistake. We'll always be slightly "behind" no matter what we do (given the turnaround time on code to being translated).

Reply:
@lieut-data Sounds good, I'll adjust the release checklist to not require cherry-picking translation PRs.

Reply:
Thanks alot @amyblais  and @lieut-data. I will update PR and remove `cherry-pick` comment on pr.

Reply:
Cherry-pick to cloud feature is removed.